src/main/resources/db/migration/V009__simuleringslager.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE simulering_lager
+(
+  id                    UUID PRIMARY KEY,
+  fagsak_id             VARCHAR NOT NULL,
+  behandling_id         VARCHAR NOT NULL,
+  fagsystem             VARCHAR NOT NULL,
+  opprettet_tidspunkt   timestamp(3) NOT NULL DEFAULT localtimestamp,
+  utbetalingsoppdrag    text NOT NULL,
```

Comment:
VARCHAR og text er det samme

Reply:
endret sql for å definere lengde for VARCHAR koloner

Reply:
Burde ikke ha med lengde i postgres, den fikser det åt oss :) Det er greit å kun bruke VARCHAR for allt med tekst

Reply:
fikset